### PR TITLE
Move code for making dependencies zip to cromwell-tools

### PR DESCRIPTION
**Pull request by dshiga**
See corresponding PR in cromwell-tools: https://github.com/broadinstitute/cromwell-tools/pull/3

### CODE REVIEW

File: lira/test/test_lira_utils.py [hunk 1]
```diff
@@ -42,29 +42,6 @@ def test_compose_inputs(self):
         self.assertEqual(inputs['foo.bundle_version'], 'baz')
         self.assertEquals(inputs['foo.runtime_environment'], 'asdf')
 
-    def test_download_to_map(self):
-        """Test download_to_map with local files to ensure it builds the map of paths to contents correctly"""
-        urls = ['data/a.txt', 'data/b.txt']
-        urls_to_content = lira_utils.download_to_map(urls)
-        self.assertIn('data/a.txt', urls_to_content)
```
Comment (samanehsan) [inline lira/test/test_lira_utils.py:49]: Can the files `a.txt` and `b.txt` be removed from lira?

**Comment by samanehsan** [comment]
I have one question about the test files, otherwise this looks good!

**Comment by dshiga** [comment]
Do you mean remove this test also, or test in a different way? The test
will break if we remove those two test data files.

File: lira/test/test_lira_utils.py (hunk 1 above)
Comment (dshiga) [reply lira/test/test_lira_utils.py:49]: oh right, yes no longer need them here

**Comment by dshiga** [comment]
I removed those test files. Back to you @samanehsan!

**Review by samanehsan — APPROVED** [review]
Thanks for making that change! This looks good to me!

**Pull request merged**